.github/workflows/preview_on_pr.yml

```diff
@@ -0,0 +1,23 @@
+name: preview-on-pr
+on: pull_request
+jobs:
+  build_and_preview:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v2
+        with:
+          repository: brown-ccv/ccv-website-nuxt
+          submodules: recursive
+          fetch-depth: 0
+      - name: Change content to current commit
+        run: cd content && git checkout ${{ github.event.pull_request.head.sha }}
+      - name: Build
+        run: npm ci && npm run generate
+      - uses: FirebaseExtended/action-hosting-deploy@v0
+        with:
+          repoToken: '${{ secrets.GITHUB_TOKEN_MIR }}'
```

Comment:
why not use the included github token on the action?

Reply:
Good catch... it was left from when DISCO was around and I think I replaced in more places than needed

Reply:
And that may fix the commenting to PR? I think that secret doesn't even exist

Reply:
@mcmcgrath13  You fixed it! 👩‍🚀